records_mover/records/targets/base.py

```diff
@@ -123,7 +123,16 @@ def can_move_from_fileobjs_source(self) -> bool:
         pass
 
 
-class SupportsMoveFromTempLocAfterFillingIt(NegotiatesRecordsFormat, metaclass=ABCMeta):
+class MightSupportMoveFromTempLocAfterFillingIt(NegotiatesRecordsFormat, metaclass=ABCMeta):
+    # TODO: Make sure Redshift and Vertica target is configured as
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker